frontend/src/components/object/ObjectTable.vue

```diff
@@ -85,18 +96,47 @@ async function showPermissions(objectId: string) {
 
 watch(getObjects, async () => {
```

Comment:
`getObjects` is the object data that's in the store. We won't need to watch it any more.